pkg/global/bun.go

```diff
@@ -57,9 +57,8 @@ func InstallBun(ctx context.Context) error {
 			isMusl = true
 		} else {
 			cmd := exec.Command("ldd", "--version")
-			if output, err := cmd.CombinedOutput(); err == nil {
-				isMusl = strings.Contains(strings.ToLower(string(output)), "musl")
-			}
+			output, _ := cmd.CombinedOutput()
+			isMusl = strings.Contains(strings.ToLower(string(output)), "musl")
```

Comment:
maybe this was changed by mistake?

```suggestion
			if output, err := cmd.CombinedOutput(); err == nil {
				isMusl = strings.Contains(strings.ToLower(string(output)), "musl")
			}
```

Reply:
As I mentioned in my original post, this was partially failing cause ldd —version didn’t return 0, at least on the alpine I tested with with, resulting in isMusl not being set, even if the stdout did in fact include musl. 

There might be a more idiomatic go way to write that, but checking for err == null was part of the problem in my test env.